### PR TITLE
Add link from main readme to installation docs

### PR DESCRIPTION
**Pull request by echlebek**
Simple QoL improvement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ following diagram provides an overview of the collection process.
 
 ## Installation
 
-Detailed instructions are available in our Installation Guides in documentation below.
+Detailed instructions are available in our [Installation Guides](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/docs/installation.md).
```
Comment (aboguszewski-sumo) [inline README.md:12]:
```suggestion
Detailed instructions are available in our [Installation Guides](/docs/installation.md).
```
Absolute repo path is better in my opinion, then it won't matter which branch the reader is on.

**Comment by swiatekm-sumo** [comment]
The idea was for the user to use one of the versioned links below, and then go to the installation guide. Given that the guide changes very little between versions, this is probably fine anyway.

**Comment by echlebek** [comment]
Oh I see... that wasn't clear to me. I can retract the PR if you prefer.

**Comment by swiatekm-sumo** [comment]
> Oh I see... that wasn't clear to me. I can retract the PR if you prefer.

I think it's a net benefit. If it wasn't obvious to you, it won't be obvious for other new users either.

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**